core/src/test/java/com/gentics/mesh/core/schema/SchemaChangesEndpointTest.java

```diff
@@ -169,6 +171,46 @@ public void testUpdateWithConflictingName() {
 		}
 	}
 
+	private SchemaUpdateRequest buildListFieldUpdateRequest(SchemaResponse schema) {
+		SchemaUpdateRequest req = new SchemaUpdateRequest();
+		req.setName(schema.getName());
+		req.setFields(schema.getFields());
+		return req;
+	}
+
+	@Test
+	public void testListTypeChange() throws Exception {
+		SchemaContainer schemaContainer = schemaContainer("content");
+		String schemaUuid = tx(() -> schemaContainer.getUuid());
+		SchemaResponse schema = call(() -> client().findSchemaByUuid(schemaUuid));
+
+		ListFieldSchema listField = new ListFieldSchemaImpl();
+		listField.setListType("string");
+		listField.setName("testListType");
+		listField.setLabel("testListTypeLabel");
+		schema.getFields().add(listField);
+		CountDownLatch latch = TestUtils.latchForMigrationCompleted(client());
```

Comment:
Use waitForJobs(Runnable action, MigrationStatus status, int expectedJobs) instead of latch.